src/test/groovy/com/netflix/gradle/plugins/deb/validation/DebPackageNameValidatorTest.groovy

```diff
@@ -0,0 +1,35 @@
+package com.netflix.gradle.plugins.deb.validation
+
+import spock.lang.Issue
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class DebPackageNameValidatorTest extends Specification {
+    DebPackageNameAttributeValidator validator = new DebPackageNameAttributeValidator()
+
+    @Issue("https://github.com/nebula-plugins/gradle-ospackage-plugin/issues/90")
+    @Unroll
+    def "verifies #description: '#attribute'"() {
+        when:
+        boolean valid = validator.validate(attribute)
+
+        then:
+        valid == result
+
+        where:
+        attribute            | result | description
+        'abc-1.0'            | true   | 'valid package name with mixed alphanumeric characters'
```

Comment:
This test case is slightly confusing to me, its unclear if the 1.0 is part of the package name or the version. Add a few more true examples.